.templates/yeet/Makefile

```diff
@@ -23,7 +23,7 @@ CFLAGS = -g -O2 -target bpf -I include
 
 SRCS := $(wildcard $(SRCDIR)/*.bpf.c)
 OBJS = $(patsubst $(SRCDIR)/%.c,$(BUILDDIR)/%.o,$(SRCS))
-CONTAINER ?= $(shell test -e /.dockerenv && echo no || echo yes)
+CONTAINER ?= $(shell test -e /.dockerenv && echo yes || echo no)
```

Comment:
Also, I don't have docker installed anywhere to check, should this be `$HOME/.dockerenv` or does docker actually put a file in `/`?